### PR TITLE
Added secondary tool icon

### PR DESCRIPTION
**Pull request by jamesdeacon**
I've made adjustments to the board.js and board.css files so that the secondary tool icon is shown in the bottom right corner of the tool button. This also switches correctly once the tool is clicked

This fixes issue https://github.com/lovasoa/whitebophir/issues/140

(This is my first ever Pull Request so please excuse me if I've done anything wrong / incorrectly but let me know so I know for future!)

<sub>
By opening a pull request, I certify that I hold the intellectual property of the code I am submitting,
and I am granting the initial authors of WBO a perpetual, worldwide, non-exclusive, royalty-free, and irrevocable license to this code.
</sub>

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1175,7 +1175,7 @@
       "handlebars": {
          "version": "4.7.6",
          "resolved": "https://registry.npmjs.org/handlebars/-/handlebars-4.7.6.tgz",
-         "integrity": "sha512-1f2BACcBfiwAfStCKZNrUCgqNZkGsAT7UM3kkYtXuLo0KnaVfjKOyf7PRzB6++aK9STyT1Pd2ZCPe3EGOXleXA==",
+         "integrity": "sha1-1MBcG6+Q6ZRfd6pop6IZqkp9904=",
```
Comment (lovasoa) [inline package-lock.json:1178]: I don't think we want that...

File: client-data/js/board.js [hunk 2]
```diff
@@ -123,6 +123,14 @@ Tools.HTML = {
...
 				toolShortcut + ")" +
 				(Tools.list[toolName].secondary ? " [" + Tools.i18n.t("click_to_toggle") + "]" : "");
+			if(Tools.list[toolName].secondary) {
+				elem.classList.add('hasSecondary');
+				var secondaryIcon = toolIconElem.cloneNode();
+				secondaryIcon.classList.add("secondaryIcon");
+				secondaryIcon.src = Tools.list[toolName].secondary.icon;
+				elem.appendChild(secondaryIcon);
+				toolIconElem.classList.add("primaryIcon");
+			}
```
Comment (lovasoa) [inline client-data/js/board.js:133]: We should probably avoid to dynamically create dom nodes here ? Can we create the img tag directly  in `board.html` and just set its src here ?

**Review by lovasoa — CHANGES_REQUESTED** [review]
Nice feature ! I made a few comments that you can already address. I'll test that on my machine when I can, and then we can merge.

File: package-lock.json (hunk 1 above)
Comment (jamesdeacon) [reply package-lock.json]: Don't know why it did that. Reverted in latest commit

File: client-data/js/board.js (hunk 2 above)
Comment (jamesdeacon) [reply client-data/js/board.js]: Yep, fair point. Made that change now

File: client-data/board.html [hunk 3]
```diff
@@ -39,7 +39,9 @@
 		<div id="menuItems">
 			<ul id="tools" class="tools">
 				<li class="tool" tabindex="-1">
-					<img class="tool-icon" width="35" height="35" src="" alt="icon" /><span class="tool-name"></span>
+					<img class="tool-icon" width="35" height="35" src="" alt="icon" />
+					<span class="tool-name"></span>
+					<img class="tool-icon secondaryIcon" width="35" height="35" src="" alt="icon" style='display:none;'/>
```
Comment (lovasoa) [inline client-data/board.html:44]:
```suggestion
					<img class="tool-icon secondaryIcon" width="35" height="35" src="" alt="icon" />
```

We can set the style in css

**Review by lovasoa — CHANGES_REQUESTED** [review]

File: package-lock.json [hunk 4]
```diff
@@ -1187,7 +1187,7 @@
             "minimist": {
                "version": "1.2.5",
                "resolved": "https://registry.npmjs.org/minimist/-/minimist-1.2.5.tgz",
-               "integrity": "sha512-FM9nNUYrRBAELZQT3xeZQ7fmMOBg6nWNmJKTcgsJeaLstP/UODVpGsr5OhXhhXg6f+qtJ8uiZ+PUxkDWcgIXLw=="
+               "integrity": "sha1-Z9ZgFLZqaoqqDAg8X9WN9OTpdgI="
```
Comment (lovasoa) [inline package-lock.json:1190]:
```suggestion
               "integrity": "sha512-FM9nNUYrRBAELZQT3xeZQ7fmMOBg6nWNmJKTcgsJeaLstP/UODVpGsr5OhXhhXg6f+qtJ8uiZ+PUxkDWcgIXLw=="
```

File: package-lock.json [hunk 5]
```diff
@@ -2780,7 +2780,7 @@
       "serve-static": {
          "version": "1.14.1",
          "resolved": "https://registry.npmjs.org/serve-static/-/serve-static-1.14.1.tgz",
-         "integrity": "sha512-JMrvUwE54emCYWlTI+hGrGv5I8dEwmco/00EvkzIIsR7MqrHonbD9pO2MOfFnpFntl7ecpZs+3mW+XbQZu9QCg==",
+         "integrity": "sha1-Zm5jbcTwEPfvKZcKiKZ0MgiYsvk=",
```
Comment (lovasoa) [inline package-lock.json:2783]:
```suggestion
         "integrity": "sha512-JMrvUwE54emCYWlTI+hGrGv5I8dEwmco/00EvkzIIsR7MqrHonbD9pO2MOfFnpFntl7ecpZs+3mW+XbQZu9QCg==",
```

**Review by lovasoa — CHANGES_REQUESTED** [review]

**Comment by lovasoa** [comment]
I made the small css fix myself. The change is now live on https://wbo.ophir.dev

Thank you for your contribution @jamesdeacon !
Feel free to make more PRs like this one 💯

**Comment by jamesdeacon** [comment]
Thanks! First ever PR! 🥳

**Pull request merged**